packages/plugins/Web3Profile/src/SNSAdaptor/components/ImageList.tsx

```diff
@@ -0,0 +1,342 @@
+                                    <ListItem
+                                        className={classes.collectionWrap}
+                                        onClick={() => unList(collection.key)}
+                                        key={collection.key}>
+                                        <NFTImageCollectibleAvatar
+                                            pluginId={NetworkPluginID.PLUGIN_EVM}
+                                            size={64}
+                                            token={{
+                                                ...collection,
+                                                tokenId: collection.tokenId ?? '',
+                                                id: collection.address,
+                                                chainId: ChainId.Mainnet,
+                                                schema: SchemaType.ERC721,
+                                                type: TokenType.NonFungible,
+                                                contract: {
+                                                    chainId: ChainId.Mainnet,
+                                                    name: collection?.name ?? '',
+                                                    symbol: '',
+                                                    address: collection.address,
+                                                    schema: SchemaType.ERC721,
+                                                },
+                                                metadata: {
+                                                    imageURL: collection.iconURL,
+                                                    chainId: ChainId.Mainnet,
+                                                    name: '',
+                                                    symbol: '',
+                                                },
+                                            }}
+                                        />
+                                    </ListItem>
+                                ))}
+                            </List>
+                        ) : (
+                            (!collectionList || collectionList?.length === 0) && (
+                                <Typography className={classes.unListedEmpty}>{t.no_items_found()}</Typography>
+                            )
+                        )}
+                    </Box>
+                    <Box>
+                        <Typography sx={{ fontSize: '16px', fontWeight: 700, padding: 2 }}>{t.unlisted()}</Typography>
+                    </Box>
+                    <Box className={classNames(classes.unlistedBox, classes.scrollBar)}>
+                        {unListedCollections && unListedCollections.length > 0 ? (
+                            <List className={classes.list}>
+                                {unListedCollections.map((collection, i) => (
+                                    <ListItem
```

Comment:
Refactor the list item as a reusable component. Or perhaps a reuseable list component.